File: ontrack-extension-git/src/main/java/net/nemerosa/ontrack/extension/git/client/plot/GDim.java
package net.nemerosa.ontrack.extension.git.client.plot;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor(access = AccessLevel.PROTECTED)
public class GDim {

    private final int w;
    private final int h;

    public static GDim of(int w, int h) {
        return new GDim(w, h);
    }
}

File: ontrack-git/src/main/java/net/nemerosa/ontrack/git/model/plot/GDim.kt
package net.nemerosa.ontrack.git.model.plot

data class GDim(
        val w: Int,
        val h: Int
) {
    companion object {
        fun of(w: Int, h: Int): GDim {
            return GDim(w, h)
        }
    }
}
